src/resilience/registration/Custom.hpp

```diff
@@ -38,47 +38,45 @@
  *
  * Questions? Contact Christian R. Trott (crtrott@sandia.gov)
  */
-#ifndef INC_RESILIENCE_CREF_HPP
-#define INC_RESILIENCE_CREF_HPP
 
-#include <vector>
+#ifndef _INC_RESILIENCE_REGISTRATION_CUSTOM_HPP
```

Comment:
Leading underscore unfortunately conflicts with the standard library (can't lead underscore with a capital letter after in C++ identifiers)